src/components/ClaimCard/ClaimCard.test.tsx

```diff
@@ -0,0 +1,40 @@
+import { render } from '@testing-library/react';
+
+import { ClaimCard, ClaimCardProps } from './ClaimCard';
+
+let props: ClaimCardProps;
+
+describe('ClaimCard', () => {
+  beforeEach(() => {
+    props = {
```

Comment:
there's no need to define this in `beforeEach`, since it's static. it can just be defined as a `const` at the top level of the file